website/static/js/jquery.homeycombs.js

```diff
@@ -0,0 +1,145 @@
+(function($) {

```

Comment:
Yes it was not available in any CDN as it is not a library. Just a small piece of jquery.